civis/ml/_model.py

```diff
@@ -857,6 +908,8 @@ def train(self, df=None, csv_path=None, table_name=None,
             train_args['FIT_PARAMS'] = json.dumps(fit_params)
         if self.dependencies:
             train_args['DEPENDENCIES'] = ' '.join(self.dependencies)
+        if n_jobs and self.train_template_id == max(_PRED_TEMPLATES.keys()):
```

Comment:
Why is this one checking `self.train_template_id == max(_PRED_TEMPLATES.keys())` instead of `_NEWEST_CIVISML_VERSION`?